### PR TITLE
Update About page

### PR DESCRIPTION
**Pull request by Erin-Mounts**
since we all work at Open Bridge Inc. now

### CODE REVIEW

**Comment by nategbrown9** [comment]
Should we replace the About Us paragraph with this from the one pager Mike shared:
`At Open Bridge Inc., we believe in a patient-centric view of health, where comprehensive, real-world data plays an essential role in understanding human health and the lived experience with disease, while improving equity in healthcare and encouraging patient engagement. Our goal is to make it easy for health researchers and clinicians to develop and deploy well-validated, unsupervised digital assessments and passive monitoring of patients, leading to improved human and AI-driven insights that improve human health.`

**Review by syoung-smallwisdom — APPROVED** [review]
👍

**Comment by Erin-Mounts** [comment]
> Should we replace the About Us paragraph with this from the one pager Mike shared: `At Open Bridge Inc., we believe in a patient-centric view of health, where comprehensive, real-world data plays an essential role in understanding human health and the lived experience with disease, while improving equity in healthcare and encouraging patient engagement. Our goal is to make it easy for health researchers and clinicians to develop and deploy well-validated, unsupervised digital assessments and passive monitoring of patients, leading to improved human and AI-driven insights that improve human health.`

Good idea.

**Review by nategbrown9 — APPROVED** [review]

**Review by DwayneJengSage — APPROVED** [review]

File: src/AboutUs.tsx [hunk 1]
```diff
@@ -201,10 +201,13 @@ const AboutUs: FunctionComponent<{onJoin: () => void}> = ({onJoin}) => {
...
-          in 2023 to develop our Digital Health platform and expertise into a 
-          turnkey service for academia and industry. Open Bridge is the result 
-          of that effort.
+        At Open Bridge Inc., we believe in a patient-centric view of health, where 
+        comprehensive, real-world data plays an essential role in understanding 
+        human health and the lived experience with disease, while improving equity 
+        in healthcare and encouraging patient engagement. Our goal is to make it 
+        easy for health researchers and clinicians to develop and deploy well-validated, 
+        unsupervised digital assessments and passive monitoring of patients, leading 
+        to improved human and AI-driven insights that improve human health.
```
Comment (syoung-smallwisdom) [inline src/AboutUs.tsx]:
```suggestion
        to human and AI-driven insights that improve human health.
```

Maybe just one "improve"?

**Pull request merged**